CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_shopify v0.14.0
+
+[PR #92](https://github.com/fivetran/dbt_shopify/pull/92) includes the following updates:
+## Under the Hood
+- Adds enable/disable config for the `metadata` staging model using the `shopify_using_metafield` variable (default `true`).
+- Adds enable/disable config for the `abandoned_checkout` staging models using the `shopify_using_abandoned_checkout` variable (default `true`):
+   - `stg_shopify__abandoned_checkout`
+   - `stg_shopify__abandoned_checkout_discount_code`
+   - `stg_shopify__abandoned_checkout_shipping_line`. 
```

Comment:
We should also callout the downstream impacts of these variables within this package. For example, the metafield variable is now a requirement in the metafield tables and the abandoned checkout variable will turn off relevant information in X end models.

Reply:
good point,updated